src/main/steps/sequence/relationship-date/template.njk

```diff
@@ -0,0 +1,6 @@
+{% extends "common/page.njk" %}
+
+{% block page_content %}
+  <h1 class="govuk-heading-l">{{ title }}</h1>
+  <p class="govuk-body">{{ line1 }}</p>
+{% endblock %}
```

Comment:
You should be able to delete this file since it will use the the default `src/main/steps/sequence/template.njk` (which is the same) if there's no template file.